src/components/screens/reclaimBalance/reclaim.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { withTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
+import Tooltip from '@toolbox/tooltip/tooltip';
+import { PrimaryButton } from '@toolbox/buttons';
+import DialogLink from '@toolbox/dialog/link';
+import AccountMigration from '@shared/accountMigration';
+import { getActiveTokenAccount } from '@utils/account';
+import styles from './index.css';
+
+const Reclaim = ({ t }) => {
+  const account = useSelector(state => getActiveTokenAccount(state));
+  const hasEnoughtBalance = parseInt(account.token?.balance, 10) > 1000000;
```

Comment:
Is this calculation needed any where else? If yes, please move it to `utils/account`